server/controllers/emptyBuildings/post.js

```diff
@@ -0,0 +1,35 @@
+const { promisify } = require('util');
+const { reportBuilding } = require('../../models/queries/emptyBuildings');
+const { schemaBuildings } = require('../../validation');
+
+const postEmptyBuilding = async (req, res, next) => {
+  try {
+    const newBuild = await schemaBuildings.validate(req.body, {
+      abortEarly: false,
+    });
+    if (req.files) {
+      const file = req.files.filename;
+      const fileName = `${Date.now()}${file.name}`;
+      newBuild.thumbnail = fileName;
+      const move = promisify(file.mv);
+      move(`./server/uploads/${fileName}`)
+        .then(filename => res.send({ statusCode: 201, data: filename }))
+        .catch(() =>
+          res.send({ statusCode: 500, error: 'image could not uploaded' }),
+        );
```

Comment:
there are multiple problems here:
1. you should append the image name to the new building after it has been moved to the uploads dir
2. you are using move `asynchronously` _(using `.then()` and `.catch()`)_ so there are cases where the `res.send` will be called multiple times _(twice)_ for the same request, so you shall use `await` and for the image to be moved successfully and then append it to the new building object and then continue with your code
3. you need to use an `else` for if the user didn't upload an image (it's optional)